Sources/XiEditor/Document.swift

```diff
@@ -96,6 +96,7 @@ class Document: NSDocument {
             xiCore.closeView(identifier: identifier)
         }
         super.close()
+        editViewController?.documentClosed()
```

Comment:
This looks alright to me but this can just simply call `postDocumentClosedNotification` right?

Reply:
Yes. Thought of giving more generic callback to EditViewController so, that it can be used for other purpose if required. But right now documentClosed() method doesn't do anything but postDocumentClosedNotification. What do you think, if it doesn't sound right I can directly invoke postDocumentClosedNotification.

Reply:
I think if that's necessary then we'll change it, but right now it shouldn't be two different method calls. Just `postDocumentClosedNotification` is good enough.

Reply:
sure

Reply:
@nangtrongvuon Could you please merge this if you are ok with the change.